sim/components/ble/SimpleWeatherService.cpp

```diff
@@ -70,6 +70,20 @@ void SimpleWeatherService::Init() {
   //ble_gatts_add_svcs(serviceDefinition);
 }
 
+void SimpleWeatherService::SetCurrentWeather(uint64_t timestamp, int16_t temperature, int iconId) {
+  SimpleWeatherService::Location cityName = (SimpleWeatherService::Location)"New York";
```

Comment:
non-blocking: doesn't look like a clean solution, but I think it works. Probably no `\0' after the last character, only at the very last entry of the character array

Reply:
Yeah I don't like this solution very much. I don't think there's currently a use for it either. I can remove it.

Reply:
Removed this, as it's unused/untested currently.